package.json

```diff
@@ -16,7 +16,7 @@
     "sinon": "^1.17.6"
   },
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js"
+    "test": "./node_modules/mocha/bin/mocha tests/js/tests/*.test.js --timeout 30000"
```

Comment:
In this case we can remove timeout from here and add a hint in developer's README in "Running JS tests" section about specifying it explicitly (`--timeout 30000`), in case of such error.

However I'm not sure if default `2000ms` will be enough with Travis for entire test suite. But we can try and increase it in case of problems though.